roles/docker/vars/redhat.yml

```diff
@@ -1,9 +1,15 @@
 docker_kernel_min_version: '0'
 
+docker_versioned_pkg:
```

Comment:
Can you include a hint comment like we use for ubuntu/debian that lets us see a list of available packages?